integration_test/get_flow_diff.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+#
+
+TEST=${1}
+DIR=log/itest/current/${TEST}
+
+LOG_DIR=${DIR}
+
+ssh root@192.168.2.91 "/opt/axsh/openvnet/ruby/bin/ruby /opt/axsh/openvnet/vnet/bin/vnflows-monitor c 15 d 1 > /tmp/itest1-1-4-flows.txt" &
```

Comment:
While it's unlikely to execute this test on other environments, I do agree that it would be a good idea to read these values from here: https://github.com/axsh/openvnet/blob/develop/integration_test/config/itest.yml